### PR TITLE
Used python block formatting and added block claim fees 

### PR DESCRIPTION
**Pull request by jack60612**
Solves #138 

Requirements:

- [x] [PR:9793](https://github.com/Chia-Network/chia-blockchain/pull/9793) Needs to be merged into main.

### CODE REVIEW

**Review by mariano54 — COMMENTED** [review]
Has this been tested?
Also, i suggest adding a comment that says:
" an announcement consumed is required for full security against unbundling attacks"

**Comment by jack60612** [comment]
> Has this been tested? Also, i suggest adding a comment that says: " an announcement consumed is required for full security against unbundling attacks"

Currently not working after the last commit. Trying to find another similar solution

**Comment by jack60612** [comment]
@mariano54 i revered it back to a working state and i will add the announcement soon

**Review by mariano54 — COMMENTED** [review]
Looks good, but it needs to be tested on testnet before merge.

File: pool/singleton.py [hunk 1]
```diff
@@ -186,6 +196,24 @@ async def create_absorb_transaction(
         #  - create an output with slightly less XCH, to yourself. for example, 1.7499 XCH
         #  - The remaining value will automatically be used as a fee
 
+    if fee_amount > 0:
+        # address can be anything
+        signed_transaction: TransactionRecord = await wallet_rpc_client.create_signed_transaction(
+            additions=[{"amount": uint64(1), "puzzle_hash": fee_target_puzzle_hash}],
+            fee=fee_amount,
```
Comment (yan74) [inline pool/singleton.py:203]: shouldn't this be ```fee=fee_amount * len(coinbase_coin_records)```?

**Comment by jack60612** [comment]
Tested and ready!

**Review by mariano54 — APPROVED** [review]
Approve for merge after corresponding chia-blochain PR is merged

**Pull request merged**